generators/service-watson-conversation/templates/python/instrumentation.py

```diff
@@ -1,10 +1,23 @@
 from ibmcloudenv import IBMCloudEnv
-from watson_developer_cloud import ConversationV1
-
-conversation = ConversationV1(
-    username=IBMCloudEnv.getString('watson_conversation_username'),
-    password=IBMCloudEnv.getString('watson_conversation_password'),
-    version='2017-05-26')
+from watson_developer_cloud import AssistantV1
 
+if IBMCloudEnv.getString('watson_conversation_apikey'):
+    iam_url = 'https://iam.stage1.bluemix.net/identity/token' if 'gateway-s.' in params['url'] else 'https://iam.bluemix.net/identity/token'
+    iam_apikey = IBMCloudEnv.getString('watson_conversation_apikey')
+    conversation = AssistantV1(
+        url=IBMCloudEnv.getString('watson_conversation_url'),
+        iam_api_key=iam_apikey,
+        version='2018-02-16',
+        iam_url=iam_url)
+else:
+    conversation = AssistantV1(
+        username=IBMCloudEnv.getString('watson_conversation_username'),
```

Comment:
Maybe we can add `url=IBMCloudEnv.getString('watson_conversation_url')` for cf instances after password.

Reply:
@alembe what do you think about this one?